### PR TITLE
Fixed DNS lookup test

### PR DESCRIPTION
**Pull request by vkresch**
---
name: Fixed DNS lookup test
about: Hotfix CI API request regression test
labels: 

---

Resolves https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/issues/421

I created a seperate PR to resolve the current CI issue.

* **Analysis**: The current CI is failing due to the removal of the carthage dns.

* **Solution**: Replace the current failing unit test which expects no response (204) with dns lookup error. Add a unit test for no response with a patch.

* **Implementation**: See solution.

* **Performed tests**: Executed the unit tests.

* **Documentation**: The test function names are self documenting.

* **Check list**:

- [ ] I extended the Github Actions CI test units with the corresponding tests for this new feature (if needed).
- [ ] I extended the Sphinx documentation (if needed).
- [ ] I extended the help section (if needed).
- [ ] I changed the README file (if needed).
- [ ] I created a new issue if there is further work left to be done (if needed).

**Work effort**: 1h

### CODE REVIEW

File: tests/regression/test_tzkt_no_content_request.py [hunk 1]
```diff
@@ -10,10 +10,39 @@ def test_request_no_content_response():
 
     # The baker address exists **only** on the mainnet
     baker_address = "tz1NortRftucvAkD1J58L32EhSVrQEWJCEnB"
-    base_url = "https://api.carthage.tzkt.io/v1/"
+    base_url = "https://api.delphinet.tzkt.io/v1/"
```
Comment (jdsika) [inline tests/regression/test_tzkt_no_content_request.py:13]: shouldn't it be something like "https://api.edo2net.tzkt.io/v1/" ? Bad to have tests that are depending on moving targets. I am not sure how long those addresses for old networks will be maintained?

Comment (vkresch) [reply tests/regression/test_tzkt_no_content_request.py:13]: The test tests if the baker which only exists on the mainnet exists on delphinet which he doesn't thus throwing the 204 response. https://api.edo2net.tzkt.io/v1/ would give an OK (200) response.

Comment (habanoz) [reply tests/regression/test_tzkt_no_content_request.py:13]: @jdsika has a point. Delphinet will close after some time. Can we use a dummy address for this purpose?

**Review by habanoz — COMMENTED** [review]
I have added a comment.

File: tests/regression/test_tzkt_no_content_request.py (hunk 1 above)
Comment (vkresch) [reply tests/regression/test_tzkt_no_content_request.py]: done

**Comment by jdsika** [comment]
@vkresch this is now rdy to be merged right?

**Comment by jdsika** [comment]
This was part of the other PR already. I am closing

**Pull request closed without merging**